modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/questions/ConnectToClusterQuestion.java

```diff
@@ -108,6 +112,25 @@ public FlowBuilder<Void, String> askQuestionIfConnected(String nodeUrl) {
         }
         return Flows.from(nodeUrl);
     }
+    /**
+     * Ask if the user wants to store credentials in config.
+     *
+     * @param username username.
+     * @param password password
+     * @return {@link FlowBuilder} instance which provides result.
+     */
+    public void askQuestionToStoreCredentials(@Nullable String username, @Nullable String password) {
+        if (!nullOrBlank(username) && !nullOrBlank(password)) {
+            QuestionUiComponent question = fromYesNoQuestion(
+                    "Do you want to store username and password in cli configuration config?"
```

Comment:
I think a question like: "Remember current credentials?" is shorter and less-implementation-specific.